tools/protodoc/protodoc.py

```diff
@@ -222,31 +227,19 @@ def FormatExtension(extension):
   """
   try:
     extension_metadata = EXTENSION_DB[extension]
-    anchor = FormatAnchor('extension_' + extension)
     status = EXTENSION_STATUS_VALUES.get(extension_metadata['status'], '')
     security_posture = EXTENSION_SECURITY_POSTURES[extension_metadata['security_posture']]
-    extension = EXTENSION_TEMPLATE.substitute(anchor=anchor,
-                                              extension=extension,
-                                              status=status,
-                                              security_posture=security_posture)
-
-    categories = FormatExtensionList(extension_metadata["categories"], "extension_category")
-    cat_or_cats = "categories" if len(categories) > 1 else "category"
-    category_message = f"This extension extends and can be used with the following extension {cat_or_cats}"
-    extension_category = EXTENSION_CATEGORIES_TEMPLATE.substitute(message=category_message,
-                                                                  categories=categories)
-    return f"{extension}\n\n{extension_category}"
+    categories = extension_metadata["categories"]
   except KeyError as e:
     sys.stderr.write(
         f"\n\nDid you forget to add '{extension}' to source/extensions/extensions_build_config.bzl?\n\n"
     )
     exit(1)  # Raising the error buries the above message in tracebacks.
 
-
-def FormatExtensionList(items, prefix="extension", indent=2):
-  indent = " " * indent
-  formatted_list = "\n".join(f"{indent}- :ref:`{ext} <{prefix}_{ext}>`" for ext in items)
-  return f"{formatted_list}\n{indent}\n"
+  return EXTENSION_TEMPLATE.render(extension=extension,
```

Comment:
There are probably a ton of other cleanup possibilities in this file. Would you prefer to leave them as TODOs?

Reply:
yep - for now i think yes - im happy to follow up tho

Reply:
i updated the PR to "partial fix for.," so we keep the sphinx ticket open once this lands